core/src/main/java/org/eclipse/krazo/core/ViewResponseFilter.java

```diff
@@ -1,246 +1,285 @@
-        entity = responseContext.getEntity();
-        if (entity != null) {
-            final String view = appendExtensionIfRequired(((Viewable) entity).getView());
-            final String uri = uriInfo.getBaseUri() + noStartingSlash(noPrefix(view, REDIRECT));
-            if (view.startsWith(REDIRECT)) {
-                responseContext.setStatusInfo(SEE_OTHER);
-                responseContext.getHeaders().putSingle(HttpHeaders.LOCATION, uri);
-                responseContext.setEntity(null);
-            }
-        }
-
-        // Fire ControllerRedirectEvent event
-        if (isEventObserved(ControllerRedirectEvent.class)) {
-            final int status = responseContext.getStatus();
-            if (status == SEE_OTHER.getStatusCode() || status == MOVED_PERMANENTLY.getStatusCode()
-                    || status == FOUND.getStatusCode() || status == TEMPORARY_REDIRECT.getStatusCode()) {
-                final ControllerRedirectEventImpl event = new ControllerRedirectEventImpl();
-                event.setUriInfo(uriInfo);
-                event.setResourceInfo(resourceInfo);
-                event.setLocation(URI.create(responseContext.getHeaderString(HttpHeaders.LOCATION)));
-                event.setContainerRequestContext(requestContext);
-                event.setContainerResponseContext(responseContext);
-                dispatcher.fire(event);
-            }
-        }
-    }
-
-    private String appendExtensionIfRequired(String viewName) {
-        return appendExtensionIfRequired(viewName, krazoConfig.getDefaultViewFileExtension());
-    }
-
-    /*
-     * Append to view name default extension if one available and applicable.
-     */
-    static String appendExtensionIfRequired(String viewName, String defaultExtension) {
-        if (viewName == null || viewName.startsWith(REDIRECT)
-                || defaultExtension == null || defaultExtension.length() == 0) {
-            return viewName;
-        }
-
-        String resultView = viewName;
-        if (!viewName.contains(".")) {
-            resultView += "." + defaultExtension;
-        }
-        return resultView;
-    }
-
-    private static MediaType selectVariant(Request request, ResourceInfo resourceInfo) {
-
-        Produces produces = resourceInfo.getResourceMethod().getAnnotation(Produces.class);
-        if (produces == null) {
-            produces = getAnnotation(resourceInfo.getResourceClass(), Produces.class);
-        }
-
-        if (produces != null) {
-
-            List<Variant> variants = Arrays.stream(produces.value())
-                    .map((String mt) -> Variant.mediaTypes(MediaType.valueOf(mt)).build().get(0))
-                    .collect(Collectors.toList());
-
-            Variant variant = request.selectVariant(variants);
-            if (variant != null) {
-                return variant.getMediaType();
-            }
-
-        }
-
-        return null;
-
-    }
-
-}
+/*

```

Comment:
For some reason the full file is marked as changed in the diff viewer. Did you perhaps replace CRLF with LF or something like that?

Reply:
will check for this one.